### PR TITLE
Allow users to convert timezone in elasticsearch module filesets

### PR DESCRIPTION
**Pull request by ycombinator**
This PR updates the following filesets in the `elasticsearch` Filebeat module to accept a `var.convert_timezone` configuration setting:

* [x] server
* [x] audit
* [x] slowlog
* [x] deprecation

The `gc` fileset was not modified because Elasticsearch GC logs already emit timestamps with timezone information in them.

Fixes partially #9756. Related: #9797.

### CODE REVIEW

**Comment by ruflin** [comment]
I think this needs a `make update` probably on the top level to also update the x-pack filebeat.

**Comment by ycombinator** [comment]
CI failures are related. Looking into them now.

**Comment by ycombinator** [comment]
jenkins, test this

**Comment by ycombinator** [comment]
@ruflin CI is green now, if you want to review the PR.

File: filebeat/module/elasticsearch/audit/test/test.log-expected.json [hunk 1]
```diff
@@ -1,107 +1,107 @@
...
-        "elasticsearch.audit.layer": "rest", 
-        "elasticsearch.audit.origin_address": "147.107.128.77", 
-        "elasticsearch.audit.principal": "i030648", 
-        "elasticsearch.audit.uri": "/_xpack/security/_authenticate", 
-        "event.dataset": "audit", 
-        "event.module": "elasticsearch", 
-        "input.type": "log", 
-        "log.offset": 0, 
-        "message": "[2018-06-19T05:16:15,549] [rest] [authentication_failed]        origin_address=[147.107.128.77], principal=[i030648], uri=[/_xpack/security/_authenticate]", 
+        "@timestamp": "2018-06-19T05:16:15.549Z",
```
Comment (ruflin) [inline filebeat/module/elasticsearch/audit/test/test.log-expected.json]: I'm surprised by the end line space diff as this file is auto generated there should not be a difference. Did you touch this file manually perhaps?

Comment (ycombinator) [reply filebeat/module/elasticsearch/audit/test/test.log-expected.json]: I did manually touch it, to replace the `@timestamp` values in each expected document.

Comment (ruflin) [reply filebeat/module/elasticsearch/audit/test/test.log-expected.json]: Why not generate it?

Comment (ycombinator) [reply filebeat/module/elasticsearch/audit/test/test.log-expected.json]: Sure, I can regenerate them. It was a small change that I could make it global search replace so I didn't bother, but I'll regenerate them now.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -202,6 +202,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...v7.0.0-alpha2[Check the
 - Rename many `icinga.*` fields to map to ECS. {pull}9294[9294]
 - Rename many `postgresql.log.*` fields to map to ECS. {pull}9303[9303]
 - Rename many `kafka.log.*` fields to map to ECS. {pull}9297[9297]
+- Add `convert_timezone` option to Elasticsearch module to convert dates to UTC. {issue}9756[9756] {pull}9761[9761]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: I think this changelog is in the wrong place as alpha2 is already shipped. Best directly add it to `.next` changelog.

File: filebeat/module/elasticsearch/audit/test/test.log-expected.json (hunk 1 above)
Comment (ycombinator) [reply filebeat/module/elasticsearch/audit/test/test.log-expected.json]: Regenerated in d9a2bac.

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (ycombinator) [reply CHANGELOG.asciidoc]: Moved in d9f5f1f.

**Comment by ycombinator** [comment]
jenkins, test this

**Pull request merged**